### PR TITLE
Complete website

### PR DESCRIPTION
**Pull request by lorenzoworx**
## Learning objectives

- Understand the concept of single page application.
- Use JavaScript to manipulate DOM elements.
- Use JavaScript events.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -4,44 +4,64 @@
...
+      <p class="nav-title">Awesome Books</p>
+      <ul class="nav-links">
+        <li class="nav-link nav-list"><a class='a-list' href='#list-section'> Book List</a></li>
+        <div class="line"></div>
+        <li class="nav-link nav-add"><a class='a-add' href='#add-section'>Add a Book</a></li>
+        <div class="line"></div>
+        <li class="nav-link nav-contact"><a class='a-contact' href='#contact-section'> Contact</a></li>
+      </ul>
+    </nav>
+    <p class="date-display"></p>
```
Comment (codepantha) [inline index.html:22]: - [ ] Please align the `date-display` <p> tag to the right as shown in the wireframe below 👇 
### Wireframe
image

### Your Design
image

**Review by codepantha — CHANGES_REQUESTED** [review]
# STATUS: CHANGES REQUIRED ♻️ 

## Hi team 👋, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!



## Highlights 🎯 
✔️ Descriptive PR
✔️ Added navigation 📌 
✔️ Used correct GitHub Flow
✔️ Good commit history
✔️ No linter errors

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
- [ ] Please consider making your app responsive on mobile devices in order to improve UX 👍 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: README.md [hunk 2]
```diff
@@ -54,9 +54,10 @@
 ### Key Features <a name="key-features"></a>
 
 <!-- > Describe between 1-3 key features of the application.-->
```
Comment (andersonlebon) [inline README.md:56]: ## [OPTIONAL]

- [ ] Kidinly remove text instructions in your README
image

**Review by andersonlebon — APPROVED** [review]
## Hi @lorenzoworx and @rivasbolinga 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

nodding-yes

## To highlight :champagne: 
- All linters are passing :heavy_check_mark: 
- Good commit history :heavy_check_mark: 
- Good PR description :heavy_check_mark: 

## Optional suggestions
- [ ] Nice job :clap:  on customizing the microverse README template with your own info, but I think the logo should be changed with yours, and if you don't have one kindly delete the microverse one in your README.
image


_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question `@andersonlebon` so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**